app/src/main/java/sample/okta/android/SampleApplication.kt

```diff
@@ -44,11 +43,9 @@ class SampleApplication : Application() {
         val oidcConfiguration = OidcConfiguration(
             clientId = BuildConfig.CLIENT_ID,
             defaultScope = SampleHelper.DEFAULT_SCOPE,
+            discoveryUrl = "${BuildConfig.ISSUER}/.well-known/openid-configuration"
```

Comment:
Does it infer the discoveryUrl if it isn't supplied?

Reply:
[According to the spec](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig), the `/.well-known/openid-configuration` URL path should be appended to the issuer URL. So in the event the discoveryUrl isn't supplied, using the issuer with that path appended to it should be sufficient.

Reply:
Should I make discoveryUrl parameter accept either the issuer or the .well-known URL in this case? I think that makes sense

Reply:
I decided to use issuer instead of discoveryUrl. OidcConfiguration used discoveryUrl previously, but I see that .well-known is always prepended by the issuer. I decided to remove having to add .well-known suffix entirely